client/src/main/webjar/authentication/authentication.html

```diff
@@ -0,0 +1,26 @@
+<div>
+    <div ng-if="!auth.isAuthenticated()">
+        <button ng-click="auth.authenticate()">
+            Login
+        </button>
+    </div>
+    <div ng-if="auth.isAuthenticated()">
+        <button ng-click="auth.logout()">
+            Logout
+        </button>
+    </div>
+    <div>
+        <span>
+            email:
```

Comment:
This is a throwaway widget.  @mckoon will take over on the client side of this after this PR is merged.